### ISSUE TITLE
Regression: Demo player invisible on iPad

### ISSUE THREAD

**Issue by sigmafn** [ruffle-rs/ruffle#14108]
### Describe the bug

Since the latest nightly the Ruffle demo player is no longer visible in Safari on my iPad. The player itself still seems to load because audio plays fine.
I'm guessing this was regressed by #14093.
![9myGS7vqsn4OMZetHquAVtQGjPI](https://github.com/ruffle-rs/ruffle/assets/107855363/8a541f4f-5ff2-41da-ac23-89a8c6e03981)

### Expected behavior

The demo player should be visible.

### Content Location

https://ruffle.rs/demo/

### Affected platform

Online demo

### Operating system

iPadOS 17.1.1

### Browser

Safari 17.1.1

**Comment by torokati44** [comment]
Yes, https://github.com/ruffle-rs/ruffle/pull/14112 fixed it, thank you for the report!
